core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -423,7 +423,24 @@ object TestUtils extends Logging {
     }
 
     result.topicId(topic).get()
-}
+  }
+
+  private def isTopicExistedAndHasSameNumPartitionsAndReplicationFactor[B <: KafkaBroker](
```

Comment:
You can make this an inner def if it's only used by one method. Also, how about `Exists` instead of `Existed`?

Reply:
> You can make this an inner def if it's only used by one method.

Ah, right, this is scala!